src/app/common/services/realtime-database.service.ts

```diff
@@ -39,8 +40,20 @@ export class RealTimeDatabaseService {
 
 	createItem(itemType: string, item: any): Promise<void> {
 		const itemPath = this.getItemPath(itemType);
-
 		let key = uuid();
+		if (itemPath === 'formConfig') {
```

Comment:
Firebase doesn't allow us to create empty record. So We initialize it with one title field